<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiPlayerWindow</class>
 <widget class="QDialog" name="MultiPlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 43, 43);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;Vs&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>20</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 43, 43);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>330</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 700 30pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gameStatus">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ffffff;&quot;&gt;Final&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerturn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;It's Player 1 Turn&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="player1name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#ffffff;&quot;&gt;Username1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#ffffff;&quot;&gt;Username2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
